starlite/app.py

```diff
@@ -132,6 +132,7 @@ def __init__(
         route_handlers: List[ControllerRouterHandler],
         static_files_config: Optional[Union[StaticFilesConfig, List[StaticFilesConfig]]] = None,
         template_config: Optional[TemplateConfig] = None,
+        security: Optional[List[Dict[str, List[str]]]] = None,
```

Comment:
```python
...
from pydantic_openapi_schema.v3_1_0.security_requirement import SecurityRequirement
...
```
```suggestion
        security: Optional[List[SecurityRequirement]] = None,
```
Should give us a cross-reference to the type in P.O.S (eek about the acronym!) where there is a good description of its semantics.

Reply:
will do.